src/pkg/caendr/caendr/models/datastore/dataset_release.py

```diff
@@ -150,6 +150,54 @@ def browser_tracks(self, val):
     self.__dict__['browser_tracks'] = val
 
 
+  def _format_props_for_ds(self):
+    props = super()._format_props_for_ds()
+    props['report_type'] = props['report_type'].name
+    return props
+
+
+  #
+  # Modify supported browser tracks
+  #
+
+  def add_browser_track(self, track) -> bool:
+    '''
+      Add the given `BrowserTrackDefault` to the list of tracks supported
+      in this release.
+
+      Returns `True` if the operation changed this object,
+      i.e. if the track was not yet supported by this release.
+    '''
+    already_exists = track['display_name'] in self['browser_tracks']
+    self['browser_tracks'].append(track['display_name'])
+    return not already_exists
+
+  def remove_browser_track(self, track) -> bool:
+    '''
+      Remove the given `BrowserTrackDefault` from the list of tracks supported
+      in this release.
+
+      Returns `True` if the operation changed this object,
+      i.e. if the track was initially supported by this release.
+    '''
+    if track['display_name'] in self['browser_tracks']:
+      self['browser_tracks'].remove(track['display_name'])
+      return True
+    return False
+
+  def set_browser_track(self, track, value: bool) -> bool:
```

Comment:
the function signature could signal the operation (mainly around naming); instead of `set_` consider `modify_` and instead of `value` use `is_visible` or similar to convey type.